scripts/helpers.sh

```diff
@@ -81,6 +86,11 @@ last_resurrect_file() {
 	echo "$(resurrect_dir)/last"
 }
 
+resurrect_pane_file() {
+	local pane_id="$1"
+	echo "$(resurrect_dir)/pane_contents-${pane_id}"
```

Comment:
I can see myself having hundred thousand `pane_contents` files in a matter of days (with tmux-continuum saving environment every 5 minutes).
Ideally we'd wanna have a new directory with a save timestamp within `resurrect_dir` for each save.

I not sure if that is easily implemented, so I'm just starting the discussion..